daemon/container.go

```diff
@@ -237,6 +237,21 @@ func (daemon *Daemon) verifyContainerSettings(hostConfig *containertypes.HostCon
 				return nil, err
 			}
 		}
+
+		// Validate the healthcheck params of Config
+		if config.Healthcheck != nil {
+			if config.Healthcheck.Interval < 0 {
```

Comment:
If that, maybe we need to use a consistent validation in the whole procedure. I think there are three places to validate:
1. client side;
2. daemon received parameters from API;
3. parameters from dockerfile when building

I think consistency is the first thing(no matter less than 0 or less than 1s), the second thing to add more specific explanation in docs. 

But currently, I am on less than 0.  😄 Open for input.